### ISSUE TITLE
Incompatible with BetterFolder Plugin

### ISSUE THREAD

**Issue by ChinatsuHS** [DiscordStyles/HorizontalServerList#40]
### Describe the bug
When BetterFolder plugin is enabled HorizontalServerList is unable to return to folders and gets stuck in the servers that are within the folder.

### To Reproduce  
download/install/enable BetterFolder plugin
install/add HorizontalServerList
Open a serverfolder in HorizontalServerList new bar

### Screenshots
![image](https://github.com/DiscordStyles/HorizontalServerList/assets/10129384/f1666296-4aef-498f-b6a8-36e7fe8510f6)
![image](https://github.com/DiscordStyles/HorizontalServerList/assets/10129384/b92b15c7-0fdd-414f-90b5-29929dfb1cf4)

### Infomation (please complete the following information)
**Discord channel**: *Stable*  
**OS**: *Windows 11 Enterprise Insider Preview build 23521.1000*  
**Mod**: *Vencord*  
**Discord language**: *English*  

### Current Solution
Disable BetterFolder plugin.

**Comment by Gibbu** [comment]
Have you tried without the Steam theme enabled?

**Comment by ChinatsuHS** [comment]
> Have you tried without the Steam theme enabled?

yes... after reporting about it to disease.. he gave me the CSS to just test without the theme enabled and the same happens.
